tests/unittests/CoreGraphics/CGPathTests.mm

```diff
@@ -871,7 +871,35 @@ static bool testSymmetricEquivalence(CGPathRef path1, CGPathRef path2) {
     CGPathRelease(containedRectangle);
 }
 
-DISABLED_TEST(CGPath, SubShapesEqualityTest) {
+TEST(CGPath, CGPathEqualsCopyTest) {
+    CGMutablePathRef thePath = CGPathCreateMutable();
+    CGFloat width = 500.0f;
+    CGFloat height = 500.0f;
+    CGFloat xstart = 50.0f;
+    CGFloat ystart = 50.0f;
+
+    CGPathMoveToPoint(thePath, NULL, xstart + .75 * width, ystart + .5 * height);
+    CGPathAddArc(thePath, NULL, xstart + .5 * width, ystart + .5 * height, .5 * height, 0, M_PI / 2, true);
+    CGPathAddArc(thePath, NULL, xstart + .5 * width, ystart + .5 * height, .5 * height, M_PI / 2, 0, true);
+    CGPathMoveToPoint(thePath, NULL, xstart + .25 * width, ystart + .5 * height);
+    CGPathAddArc(thePath, NULL, xstart + .375 * width, ystart + .5 * height, .25 * height, M_PI, 0, false);
+    CGPathMoveToPoint(thePath, NULL, xstart + .5 * width, ystart + .5 * height);
+    CGPathAddArc(thePath, NULL, xstart + .625 * width, ystart + .5 * height, .25 * height, M_PI, 0, true);
+    CGPathMoveToPoint(thePath, NULL, xstart + .4375 * width, ystart + .5 * height);
+    CGPathAddArc(thePath, NULL, xstart + .375 * width, ystart + .5 * height, .125 * height, 0, M_PI / 2, true);
+    CGPathAddArc(thePath, NULL, xstart + .375 * width, ystart + .5 * height, .125 * height, M_PI / 2, 0, true);
+    CGPathMoveToPoint(thePath, NULL, xstart + .6875 * width, ystart + .5 * height);
+    CGPathAddArc(thePath, NULL, xstart + .625 * width, ystart + .5 * height, .125 * height, 0, M_PI / 2, true);
+    CGPathAddArc(thePath, NULL, xstart + .625 * width, ystart + .5 * height, .125 * height, M_PI / 2, 0, true);
+
+    CGMutablePathRef thePathCopy = CGPathCreateCopy(thePath);
+    EXPECT_TRUE(testSymmetricEquivalence(thePath, CGPathCreateCopy(thePath)));
```

Comment:
Woops, reasons to not get distracted by email pop ups.

---
In reply to: [97888196](https://github.com/Microsoft/WinObjC/pull/1813#discussion_r97888196) [](ancestors = 97888196)